csrc/quantization/aqlm/aqlm_cuda_kernel.cu

```diff
@@ -0,0 +1,215 @@
+// Adapted from https://github.com/Vahe1994/AQLM
+
```

Comment:
Ditto here, though in this case I have no modifications

Reply:
@mgoin ?  @robertgshaw2-neuralmagic

Reply:
yup